src/constants/common.js

```diff
@@ -206,9 +213,20 @@ module.exports = {
 	ROLLOUT_STATUS_ROLLED_OUT: 'ROLLED_OUT',
 	ROLLOUT_STATUS_INACTIVE: 'INACTIVE',
 	ROLL_OUT: 'rollout',
+	ROLL_OUT_MODEL: 'rollouts',
 	ROLLOUT_TYPE_PROGRAM: 'program',
 	ROLLOUT_TYPE_SOLUTION: 'solution',
 	ROLL_OUT_PATH: 'rollout/',
 	PROJECT_UPLOAD_FILE_NAME: 'project.json',
 	ROLLOUT_UPLOAD_FILE_NAME: 'rollout.json',
+	SUFFIX_CHILD: '_CHILD',
+	SOLUTIONS_RESOURCE_TYPE: {
+		project: 'Improvement Project Solution',
+	},
+	SOLUTIONS_TYPE: {
+		project: 'improvementProject',
+	},
+	ROLLOUT_STATUS_PUBLISHED: 'ROLLED_OUT',
+	ROLLOUT_PROCESS_TYPE_CREATE: 'create',
```

Comment:
@adithyadinesh0412  do we require ROLLOUT_PROCESS_TYPE_CREATE and  ROLLOUT_PROCESS_TYPE_UPDATE? Can we identify based on published_id?